README.md

```diff
@@ -29,12 +29,31 @@ w2w ./sample_data lcz_zaragoza.tif geo_em.d04.nc
 
 3. Deploy using your own data:
 ```sh
-w2w INPUT_DIRECTORY YOUR_LCZ.TIF YOUR_GEO_EM.d0X.NC
+w2w INPUT_DIRECTORY YOUR_LCZ.TIF YOUR_GEO_EM.dXX.NC
 ```
+Output
+-------
+
+   The **original files are not modified**. Three new files are generated by `w2w`:
+   - geo_em.d04_NoUrban.nc: Urban areas are replaced with the dominant surrounding rural land use.
+   - geo_em.d04_LCZ_extent.nc: The extension of urban areas is updated using LCZ.
+    - geo_em.d04_LCZ_params.nc: Urban areas are described using three different urban density categories.
+
+   Differences between the original file and the newly created files by `w2w` are shown below for the sample data provided.
+
+   <p>
+<img src = "./sample_data/W2W-BeforeAfter.png">
+</p>
+<p>
+Fig.1: W2W results on sample data. Land use categories from the original file (a), the params file (b), the extent file (d) and the NoUrban (f). Pixels replaced with respect to the original file in params (c), extent (e) and NoUrb (g).
+</p>
+
+   A [sample Jupyter notebook](./sample_data/Plotting_W2W-BeforeAfter.ipynb) is provided to recreate this Figure but is not part of the package, thus additional modules are required to run it.
```

Comment:
same here, maybe let's call it `plotting_w2w_before_after.ipynb` sticking to all lower case and snake_case